src/components/Discover/AvailableMedia/index.tsx

```
@@ -0,0 +1,105 @@
+import Header from '@app/components/Common/Header';
+import LoadingSpinner from '@app/components/Common/LoadingSpinner';
+import PageTitle from '@app/components/Common/PageTitle';
+import { sliderTitles } from '@app/components/Discover/constants';
+import TitleCard from '@app/components/TitleCard';
+import TmdbTitleCard from '@app/components/TitleCard/TmdbTitleCard';
+import useVerticalScroll from '@app/hooks/useVerticalScroll';
+import globalMessages from '@app/i18n/globalMessages';
+import Error from '@app/pages/_error';
```

Biome (2.4.6)

[error] Do not shadow the global "Error" property.

(lint/suspicious/noShadowRestrictedNames)